admin/templates/base.html

```diff
@@ -0,0 +1,109 @@
+{% load staticfiles %}
+<script src="{% static 'vendor/bower_components/raven-js/dist/raven.min.js' %}"></script>
```

Comment:
Django load commands should always be at the top of the file, outside the html. These script tags should be inside the html, right before the closing head tag, about line 29.